charts/redpanda/configmap.tpl.go

```diff
@@ -19,18 +19,767 @@ package redpanda
 import (
 	"fmt"
 
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+
 	"github.com/redpanda-data/helm-charts/pkg/gotohelm/helmette"
 )
 
+func ConfigMap(dot *helmette.Dot) []corev1.ConfigMap {
```

Comment:
Did not realize we had multiple configmaps being generated. Could we rename this function to `ConfigMaps` or possibly split it into two functions for each individual configmap?

Reply:
I will split it.